scripts/set_plugin_version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PLUGIN_FILE_PATH="../crates/snforge-scarb-plugin/Scarb.toml"
+
+VERSION=$(grep '^version' "$PLUGIN_FILE_PATH" | sed -E 's/version = "([^"]*)"/\1/')
+
+STD_FILE_PATH="../snforge_std/Scarb.toml"
+
+sed -i.bak "/snforge_scarb_plugin/ s/\(snforge_scarb_plugin = \).*/\1\"^${VERSION}\"/" $STD_FILE_PATH
+
+rm ${STD_FILE_PATH}.bak 2> /dev/null
```

Comment:
If we are always removing the `.bak` file anyway, why not just omit creating it?